errs/example_errs_test.go

```diff
@@ -9,7 +9,7 @@ import (
 )
 
 func ExampleErrorf() {
-	errInternal := errors.New("internal error")
+	const errInternal = errs.Const("internal error")
```

Comment:
iiuc this now also makes the following true, not sure if this is a good thing (to be added to assertions below, which I can't in this review):
```go
fmt.Println("err is errInternal-like:", errors.Is(err, errs.Const("internal error")))
```